sensor/common/compliance/multiplexer.go

```diff
@@ -0,0 +1,82 @@
+package compliance
+
+import (
+	"github.com/stackrox/rox/generated/internalapi/central"
+	"github.com/stackrox/rox/pkg/centralsensor"
+	"github.com/stackrox/rox/pkg/channelmultiplexer"
+	"github.com/stackrox/rox/sensor/common"
+)
+
+// Multiplexer is a wrapper around pkg.channelmultiplexer that turns it into a sensor component.
+// This is necessary since multiplexers are also used elsewhere, eg. compliance
+type Multiplexer struct {
+	mp         channelmultiplexer.ChannelMultiplexer[common.MessageToComplianceWithAddress]
+	components []common.ComplianceComponent
+}
+
+// NewMultiplexer creates a Multiplexer of type T, wrapped up as a sensor component
+func NewMultiplexer() *Multiplexer {
+	multiplexer := Multiplexer{
+		mp:         *channelmultiplexer.NewMultiplexer[common.MessageToComplianceWithAddress](),
+		components: []common.ComplianceComponent{},
+	}
+	return &multiplexer
+}
+
+// Notify is unimplemented, part of the component interface
+func (c *Multiplexer) Notify(_ common.SensorComponentEvent) {
+	// unimplemented
+}
+
+// Start starts the Multiplexer. It is important not to call this before all addChannel calls
+func (c *Multiplexer) Start() error {
+	log.Debugf("Multiplexer starts")
```

Comment:
Remove this debug message

Reply:
Is this still open or is there maybe a push missing?

Reply:
Must've overlooked this one, yeah. I removed it now.